c2corg_api/views/image.py

```diff
@@ -127,3 +146,19 @@ def put(self):
                                          self.request.authenticated_userid):
                 raise HTTPForbidden('No permission to change this image')
         return self._put(Image, schema_image)
+
+
+@resource(collection_path='/images/list', path='/images/list/{id}',
```

Comment:
Can't you set `path` to `/images/list`? Using `/images/list/{id}` might be misleading.